engines/python/setup/djl_python/rolling_batch/rolling_batch.py

```diff
@@ -407,19 +410,14 @@ def try_catch_handling(self, *args, **kwargs):
             return func(self, *args, **kwargs)
         except Exception:
             logging.exception("Rolling batch inference error")
-            err = {
-                "data": "",
-                "last": True,
-                "step_token_num": 0,
-                "code": 424,
-                "error": ""
-            }
-            results = []
-            for i in range(
-                    len(self.active_requests) + len(self.pending_requests)):
-                results.append(err)
+            for request in self.active_requests:
+                token = Token([-1], "", -1, None)
+                request.set_next_token(token,
+                                       last_token=True,
+                                       finish_reason="error")
```

Comment:
could we also add `repr(exception)` to the finish reason? something like `finish_reason=f"error: repr(e)" to let the users know what the error is?

Reply:
for some customers, they do not want to expose error log in stack traces, those info can be used by hacker to intercept the content inside